tensorboard/components/analytics.html

```diff
@@ -16,3 +16,9 @@
 -->
 
 <!-- TODO(@jart): Give users the ability to opt-in to analytics. -->
+<script>
+  // We fake the global 'ga' object, so the object is a noop. The
+  // google.analytics typing gives the object a type of UniversalAnalytics.ga.
+  // We do not track open source users.
+  window['ga'] = function() {};
```

Comment:
Why does this line not prevent analytics for working for Google-internal users? Do you remove this in a copybara pass?

Reply:
In the internal repo, this file is named `analytics.html.OPENSOURCE`. The `analytics.html` file actually has the tracking code. We just swap the filenames as part of the import process.

Reply:
Got it; thanks.